Makefile

```diff
@@ -3,7 +3,7 @@
 REPO_NAME = go-plugins-helpers
 REPO_OWNER = docker
 PKG_NAME = github.com/${REPO_OWNER}/${REPO_NAME}
-IMAGE = golang:1.7
+IMAGE = golang:1.8
```

Comment:
:sweat_smile: 
We may even bump this higher (1.10), but let's do that in a follow-up PR :wink: